Rakefile

```diff
@@ -12,10 +12,24 @@ require 'latest_ruby'
 require 'rake/clean'
 require "#{direc}/lib/#{PROJECT_NAME}/version"
 
+desc "start pry with fixture c file"
+task :pry_fixture do
+  sh %{pry -I./lib -r pry-doc -e "Pry::CInternals::CodeFetcher.ruby_source_folder = './spec/fixtures/c_source'"}
```

Comment:
Is this task temporary? Seems to be very brittle because we eval code, so when we rename something here, it'll be broken. I guess it's no biggie, though.

Reply:
@kyrylo it's brittle but ic an't think of any other way of doing it